jekyll/_cci2/configuration-reference.md

```diff
@@ -493,6 +495,20 @@ jobs:
       ... // other config
 ```
 
+The Orb is a wrapper around the `machine` executor to make configuration easier. This is the recommended
+way of handling it, but it can be directly defined as well.
```

Comment:
This makes sense. I would re-word it a little bit; something akin to:

> This is the recommended way to set up the Windows executor. It is also possible to directly define a Windows `machine image` as seen below.

Reply:
That revised wording does make a bit more sense, implemented!